src/types/Status.ts

```diff
@@ -0,0 +1,5 @@
+export enum Status {
+  all = 'all',
```

Comment:
Very cosmetic, but values should be uppercase - you can change it when rendering the value to select option to be first letter capital.